docs/source/gbd2019_models/risk_effects/zinc_deficiency/index.rst

```diff
@@ -0,0 +1,254 @@
+   :widths: 5 5 5 5 5
+   :header-rows: 1
+
+   * - Outcome
+     - Outcome type
+     - Outcome ID
+     - Affected measure
+     - Note
+   * - Diarrheal diseases
+     - Cause
+     - 302
+     - Incidence rate
+     - 
+
+Diarrheal diseases incidence rate
++++++++++++++++++++++++++++++++++
+
+Information on the diarrheal diseases cause model can be found on the :ref:`2017 Diarrheal Diseases Cause Model Document <2017_cause_diarrhea>`. 
+
+.. note::
+
+  The cause model document linked above is for the 2017 cause model document. This should be updated to the GBD 2019 cause model document when it is created.
+
+The zinc deficiency risk factor affects both the likelihood of morbidity and mortality from diarrheal diseases. We will model this in Vivarium such that exposure to zinc deficiency will impact the incidence rate of diarrheal diseases while the excess mortality rate will be unaffected.
+
+.. list-table:: GBD 2019 Risk Outcome Pair Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - Age group start
+     - 5
+     - 1-4 years 
+   * - Age group end
+     - 5
+     - 1-4 years
+
+.. note::
+
+  The zinc deficiency risk factor should affect the diarrheal incidence rate of 1 to 4 year old simulants (age_group_id=5) **only**.
+
+  There are PAFs available for age_group_id=4 using the code snippet above, although there are no relative risks or exposure data for this age group. The zinc deficiency relative risk is restricted to age_group_id=5 only, so the PAF data for age_group_id=4 should not be used in the Vivarium model.
+
+
+Therefore, the diarrheal diseases incidence rate for a given simulant should be modeled as follows:
+
+.. math::
+
+  incidence_i = incidence_\text{c302} * (1 - PAF_\text{r97,c302}) * RR_\text{r97,c302}(i)
+
+Where,
+
+.. list-table:: Parameter Definitions
+   :header-rows: 1
+
+   * - Parameter
+     - Definition
+     - Value
+     - Note
+   * - :math:`incidence_i`
+     - Diarrheal disease incidence rate for a given simulant
+     - See equation above
+     - 
+   * - :math:`incidence_\text{c302}`
+     - Diarrheal diseases incidence rate for a given age-/sex-/year-/location-specific demographic group
+     - Defined in the :ref:`2017 Diarrheal Diseases Cause Model Document <2017_cause_diarrhea>`
+     - 
+   * - :math:`PAF_\text{r97,c302}`
+     - PAF for the zinc deficiency and diarrheal diseases risk outcome pair for a given age-/sex-/year-/location-specific demographic group
+     - GBD 2019: see code snippet below
+     - 
+   * - :math:`RR_\text{r97,c302}(i)`
+     - Relative risk of diarrheal disease incidence for a given simulant based on their zinc exposure value
+     - GBD 2019: see code snippet below
+     - 
+
+PAFs and relative risks can be pulled from GBD using the code below after specifying desired sex, age_group, and location IDs.
```

Comment:
It's unclear what you mean by "using the code below after specifying desired sex, age_group, and location IDs". Since you're not specifying sex, age group, or location in the code below, won't the code pull the data for all available sexes, age groups, and locations?  I always forget exactly how get_draws works...